test/bdd/features/verifiable_credential.feature

```diff
@@ -11,3 +11,8 @@ Feature: Issue Verifiable Credential
   Scenario: Issue University Degree Credential with JWS Linked Data proof
     When "Stanford University" issues credential at "2018-03-15" regarding "Bachelor Degree" to "Alice" with "JWS Linked data" proof
     Then "Alice" receives the credential and verifies it
+
+  @issue_vc_jws
+  Scenario: Issue University Degree Credential with JWS Linked Data proof
```

Comment:
Looks like this scenario describes Issue Credential with JWS proof rather than Linked Data (line 16 and line 11 are same).

Reply:
ah, right! fixed